passflip/arguments.py

```diff
@@ -15,10 +15,15 @@ def init_arguments(self):
             help="prompt for password & salt twice."
         )
         self.add_argument(
+            "-m",
+            "--multiple",
+            action="store_true",
+            help="prompt forever until input is blank"
+        )
+        self.add_argument(
```

Comment:
Looks good. I'm quite happy with how I chose to put the command line arguments in one place. It worked out well for new ones being added.